source/addonHandler/__init__.py

```diff
@@ -566,17 +584,24 @@ def addToPackagePath(self, package):
 		self._extendedPackages.add(package)
 		log.debug("Addon %s added to %s package path", self.manifest['name'], package.__name__)
 
+	@property
+	def _canBeEnabled(self) -> bool:
+		return (
+			isAddonCompatible(self)  # Incompatible add-ons cannot be enabled

```

Comment:
I think this should use the newer syntax

```suggestion
			self.isCompatible  # Incompatible add-ons cannot be enabled
```

Reply:
Fixed in 3ef33fb20